deliverables/README.md

```diff
@@ -20,49 +20,89 @@ I seek the Rails
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
-#### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+#### Required - Answer these questions in the deliverables file, and complete the following.
+<br>
+**What is the role of HTML in a web page?**<br>
+  HTML, or HyperText Markup Language, is the skeleton/framework of the webpage. In other words, it is basically the text/content that the page consists of. 
+<br>
+**What is the role of CSS in a web page?**<br>
+  CSS, or Cascade Style Sheets, is the formatting for the webpage. This includes the elements, divs, colors, typeface, font sizes, font colors, etc. 
+<br>
+**What is the role of Javascript in a web page?**<br>
+JS is a language that allows for interactive features on a webpage.
+<br>
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+**Why do we separate HTML and CSS?**<br>
+Keeping the languages seperate allows for the code to be more manageable, as well as more efficient and simpler to understand/apply changes when necessary.
+<br>
 
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+**What are classes and IDs (and how are they different)?**<br>
+Both Classes and IDs are used when styling HTML markup. They are different in that classes define the characteristics of an object in HTML. Classes can be linked to various objects, allowing for them <strong>all</strong> to have consistent formatting. IDs are used for specific, individual elements, rather than various ones. <br>
+
+**What are elements?**<br>
+An HTML element is any content that is bound by a start tag and an end tag.<br>
+**What are tags?**<br>
+Tags are commands, basically, that bind an element. Tags are held between "less than" and "greater than" symbols, and mark the beginning and the end of an element. <br>
+**What are attributes?**<br>
+Attributes provide additional information about an element and are contained in the start tag. <br>
+**What are forms?**<br>
+HTML forms are used to collect user input, and define an HTML form. <br>
```

Comment:
Circular definition here.  You could say they're used "to define a POST request to a remote server".  When you click "submit" on a form, the browser looks at the `form`'s `action` attribute and fires a POST request to that URL, with the form contents contained in the header of the request.